pyproject.toml

```diff
@@ -1,36 +1,36 @@
-[tool.poetry]
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
 name = "pyblu"
 version = "2.0.2"
 description = ""
-authors = ["Louis Christ <mail@louischrist.de>"]
-repository = "https://github.com/LouisChrist/pyblu"
-license = "MIT"
```

Comment:
@LouisChrist Just saw that this change removed the license metadata. I believe it would be better if this is reverted. `MIT` is the correct license expression I believe.
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files